Doc/library/time.rst

```diff
@@ -774,6 +774,16 @@ These constants are used as parameters for :func:`clock_getres` and
 
    .. versionadded:: 3.7
 
+.. data:: CLOCK_TAI
+
+   International atomic time, as seconds from the unix epoch.
+
+   Warning: your system must have a current leap second table in order for this to give the correct answer.
+   You can configure ptp or ntp to maintain a leap second table.
```

Comment:
Let's remove usage of the second person here. "The system must"....

Also, delete the "Warning:" prefix.